configure.ac

```diff
@@ -216,6 +211,10 @@ fi
 
 AM_CONDITIONAL(ENABLE_MAN, test x$enable_man != xno)
 
+AC_CONFIG_LINKS([
+  cli/all-tests:cli/all-tests
+  cli/fast-tests:cli/fast-tests
+])
 AC_CONFIG_FILES(
 Makefile
```

Comment:
@dbry I'd like to remove the recursive Make build system and [move to a non-recursive one](https://autotools.io/automake/nonrecursive.html). Would you accept a PR to this end?